javascript/src/config_checker.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+    parseProperties: function (properties_json) {
+        let properties;
+        let properties_abs_path = path.resolve(properties_json);
+        try {
+            properties = require(properties_abs_path);
+        } catch (err) {
+            if (err instanceof SyntaxError) {
+                throw new Error(`${properties_json} does not follow JSON syntax`);
+            } else {
+                throw new Error(`File ${properties_json} not found`);
+            }
+        }
+        if (!("methodProperties" in properties)) {
+            throw new Error(`The methodProperties property is missing in ${properties_json}`);
+        } else {
+            var missingMethod = properties.methodProperties.find(prop => !("methodName", "flagType", "argumentIndex" in prop));
+            missingMethod = JSON.stringify(missingMethod);
+
+            if (missingMethod != null) {
+                throw new Error(`${missingMethod} in ${properties_json} doesn't have all required keys`);
```

Comment:
If possible, let's report which key is missing. That said, this can easily be a follow up fix 🙂

Reply:
This was pointed out by @mkr-plse as well. We seem to have missed incorporating the change while merging a PR in our working repo. Will do it now. Thanks for pointing it out.

Reply:
Handled in [329ed90](https://github.com/uber/piranha/pull/81/commits/329ed90f3a53b13a68fa53b97a028aafe6c72abe)